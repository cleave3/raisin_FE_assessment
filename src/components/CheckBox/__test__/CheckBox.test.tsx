/**
 * @jest-environment jsdom
 */
import { cleanup, render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Checkbox from "..";

afterEach(cleanup);

describe("Checkbox", () => {
    it("It renders correctly", () => {
        const { container } = render(<Checkbox />);
        expect(container.querySelector("label")).toBeTruthy();
        expect(container.querySelector(".app-checkbox-label")).toBeTruthy();
        expect(container.querySelector(".app-checkbox")).toBeInTheDocument();
    });
});
