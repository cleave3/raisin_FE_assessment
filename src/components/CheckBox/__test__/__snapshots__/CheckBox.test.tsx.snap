// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox It renders correctly 1`] = `
<div>
  <div
    class="app-checkbox-container"
  >
    <input
      class="app-checkbox"
      id=":r0:"
      type="checkbox"
    />
    <label
      class="app-checkbox-label"
      for=":r0:"
    />
  </div>
</div>
`;
